csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/NugetPackageRestorer.cs

```diff
@@ -604,6 +604,10 @@ private bool IsFeedReachable(string feed, int timeoutMilliSeconds, int tryCount,
                 {
                     httpClientHandler.ServerCertificateCustomValidationCallback = (message, cert, chain, _) =>
                     {
+                        if (chain is null || cert is null)
+                        {
+                            return false;
+                        }
```

Comment:
Should we log something in this case? Do we expect any of these to be null in any circumstance?

Reply:
Good question, I am not really familiar with this API. This was the minimal change I could make that just replaced a potential crash with a failed validation instead.
@mbg : Do you have an opinion about it?

Reply:
Ok, that indicates that we should add a log message in case these are null.

Reply:
Yep, I was just about to add that I guess logging it as @tamasvajk suggested would make sense.